portfolio/src/main/webapp/script.js

```diff
@@ -33,12 +33,12 @@ function addRandomGreeting() {
 function testFetch() {
     console.log("Test fetch from data")
 
-    fetch('/data').then(response => response.json()).then((data) => {
-        const container = document.getElementById('greeting-container')
+    fetch('/data').then(response => response.json()).then((comments) => {
+        const container = document.getElementById('comment-container')
         container.innerText = '';
-        container.appendChild(createListElement(data[0]))
-        container.appendChild(createListElement(data[1]))
-        container.appendChild(createListElement(data[2]))
+        for(c in comments) {
```

Comment:
I would avoid "in" when iterating an array. "for (const c of comments)"

See also: https://stackoverflow.com/questions/500504/why-is-using-for-in-with-array-iteration-a-bad-idea

Reply:
Woah, had no idea this would be the case. It's quite a new domain eh? Thanks for this!

Reply:
I've been just learning JS. Basically, list of things that have bitten me in the last month...